File: commimg/hazard_classifier.js
// first you need to install something
// npm install --save watson-developer-cloud

// Then you can classify the image
var VisualRecognitionV3 = require('watson-developer-cloud/visual-recognition/v3');
var fs = require('fs');

var visualRecognition = new VisualRecognitionV3({
	version: '2019-11-17',
	iam_apikey: 'ThQOWETYg4cg7kpE-xTEfoHVbh627RxdDCNs5YrZxCAG'
});

var images_file= fs.createReadStream('./Test/yellow.jpg'); // Here you put in the picture
var classifier_ids = ["BuildingHazardClassifier_1958782173"];
var threshold = 0.6;

var params = {
	images_file: images_file,
	classifier_ids: classifier_ids,
	threshold: threshold
};

visualRecognition.classify(params, function(err, response) {
	if (err) { 
		console.log(err);
	} else {
		// console.log(JSON.stringify(response.images[0].classifiers[0].classes[0]))
	}
});

File: commimg/hazard_classifier.py
from watson_developer_cloud import VisualRecognitionV3

visual_recognition = VisualRecognitionV3(
    version='2016-05-17',
    iam_apikey="ThQOWETYg4cg7kpE-xTEfoHVbh627RxdDCNs5YrZxCAG"
)
import json
from watson_developer_cloud import VisualRecognitionV3

visual_recognition = VisualRecognitionV3(
    '2016-05-17',
    iam_apikey="ThQOWETYg4cg7kpE-xTEfoHVbh627RxdDCNs5YrZxCAG")

with open('C:/Users/Think/IBM/yellow.jpg', 'rb') as images_file:
    classes = visual_recognition.classify(
        images_file,
        threshold='0.6',
	classifier_ids='BuildingHazardClassifier_1958782173').get_result()
#print(json.dumps(classes, indent=2))
a = classes.get('images')[0].get('classifiers')[0].get('classes')